frontend/src/app/shared/error/error.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input } from '@angular/core';
+import { FormGroup, FormControl, FormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatIconModule } from '@angular/material/icon';
+import { TranslateModule } from '@ngx-translate/core';
+
+@Component({
+  selector: 'app-error',
+  templateUrl: './error.component.html',
+  imports: [
+    FormsModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatIconModule,
+    TranslateModule
+  ],
+  styleUrls: ['./error.component.css'],
+  standalone: true
+})
+export class ErrorComponent {
+  @Input() form!: FormGroup;
+
+  @Input() maxErrors: number = Number.MAX_SAFE_INTEGER;
+
+  @Input() formControlPath!: string;
```

Comment:
you can inject the mat form field to get to the control the component is located in